pallets/currency/src/mock.rs

```diff
@@ -104,11 +102,14 @@ impl pallet_balances::Config for Test {
 parameter_types! {
 	pub const GetNativeCurrencyId: CurrencyId = DEFAULT_NATIVE_CURRENCY;
 	pub const GetRelayChainCurrencyId: CurrencyId = DEFAULT_COLLATERAL_CURRENCY;
-	#[cfg(feature = "runtime-benchmarks")]
```

Comment:
this causes error to building/testing other features. 
for example: 
```
cargo +nightly test -p vault --features integration-test`
``` 
actually fails with a duplicate  lang item issue.

`cfg` macros should be done _outside_ of `parameter_types!` macros.